### DISCUSSION TITLE
I get denied when I provide sufficient proof

### DISCUSSION THREAD

**Discussion by resslr** [community/community#57561] [GitHub Education] [answered]
### Hi everyone,

I have applied using a valid school id, an official letter from my school's it department granting authorization, AND my official enrollment information from the school panel, and still have been denied, and I have been getting the same errors but due to privacy reasons and many other reasons I do not feel comfortable putting my real name in my profile, and this is limiting my ability to receive a pack I am trying to get so I can better my education further and improve my skillset so that I can become a better software engineer in the future.

### A suggestion...(optional)

Please do more thorough reviews of applications

**Comment by resslr** [comment]
*Note: I used my school provided email address also

**Comment by chriskthomas** [comment, marked as answer]
The approval is often instant. So, you might be able to change it to your real name for a few minutes and then change it back.
